assets/component-facets.css

```diff
@@ -272,11 +272,40 @@
   padding: 0.5rem 2rem;
 }
 
+.facets-layout-grid {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  text-align: center;
+  padding: 2rem 2.4rem;
+}
+
+.facets-layout-grid.facets__list--vertical {
+  padding: 1rem 0;
+}
+
+.facets-layout-grid.facets-layout-grid--swatch {
+  gap: 3rem 1rem;
+}
+
+.facets-layout-grid.facets-layout-grid--thumbnail {
+  gap: 1.2rem;
+}
+
 .facets__item {
   display: flex;
   align-items: center;
 }
 
+/* Hover/focus state */
+.facets-layout-list .facets__label:hover .facet-checkbox__text,
+.facets-layout-list input:focus ~ .facet-checkbox__text {
+  text-decoration: underline;
+}
+
+.facets-layout-grid > * {
+  align-items: flex-start;
+}
+
 .facets__item label,
```

Comment:
When showing multiple colors, it's showing as white. Is that expected? https://screenshot.click/12-00-qb4vh-ppeei.mp4

Reply:
That's because when we set up the filter, we set the override to Image (which doesn't look like you're using).

![image](https://github.com/Shopify/dawn/assets/1019769/78034570-e381-4639-b5e4-f4c76a9bc2ec)

Instead, you can click the pencil/edit icon and change Swatch override to Colors and save.